fastapi-google-stt/app/services/openai_vector_store.py

```
@@ -27,15 +27,15 @@
 # 텍스트를 임베딩하고 DB에 저장하는 함수
 async def add_text(text: str, language_code: str, script_id:str):
     now = datetime.now().strftime("%Y-%m-%d %H:%M:%S") # 현재 시간
-
     doc = Document(page_content=text, metadata={
         "script_id": script_id,
         "lang": language_code,
         "source": "stt",
         "created_at":now
     })    # 문서 객체로 변환
+    vectordb = init_vectordb()
```

Ruff (0.8.2)

Undefined name `init_vectordb`

(F821)